src/App.vue

```diff
@@ -11,7 +13,10 @@
  * Root component
  */
 import LoadingProgress from '@/components/Layout/LoadingProgress'
```

Comment:
Better use absolute instead of relative imports, and "@" as alias for "src/" (see line above)